modules/Emails/include/ComposeView/EmailsComposeView.js

```diff
@@ -1366,6 +1366,41 @@
     });
   };
 
+  $.fn.EmailsComposeView.onTemplateChange = function (args) {
```

Comment:
Hi @ojs87 thank you for your PR. Is there a chance that `onTemplateChange` is called when `onTemplateSelect` is also called?

Reply:
Hi @clemente-raposo I don't believe this is the case, I tested and it seems that since the field is not being typed in/manipulated by the user, when selecting a template from the popup, the onchange event doesn't occur.

Reply:
Thank you @ojs87